app/src/main/java/com/example/android/guesstheword/screens/game/GameViewModel.kt

```diff
@@ -7,14 +7,29 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.Transformations
 import androidx.lifecycle.ViewModel
 
+private val CORRECT_BUZZ_PATTERN = longArrayOf(100, 100, 100, 100, 100, 100)
```

Comment:
I was doing like in the tutorial and I think that the reason why these values were declared outside of the class is because they were meant to be used as global variables and they can be seen as constants. 
P.S. : I moved them in companion object.